### PR TITLE
Jupyter and Singularity

### PR DESCRIPTION
**Pull request by guacke**
Adds info on how access Jupyter notebooks running in remote Singularity containers from your local browser.

### CODE REVIEW

File: site/docs/user_documentation/getting_started.md [hunk 1]
```diff
@@ -58,3 +58,52 @@ Or copy and paste one of these URLs:
...
+```
+
+then you can pull and convert the Docker image in the scratch space
+
+```bash
+SINGULARITY_CACHEDIR=$VSC_SCRATCH/containers/cache SINGULARITY_TMPDIR=$VSC_SCRATCH/containers/tmp SINGULARITY_PULLFOLDER=$VSC_SCRATCH/containers singularity pull docker://gqcg/gqcp
+```
+
+
+Note that the above `SINGULARITY_*` environment variables can also be set user wide in your `.bashrc`.
```
Comment (lelemmen) [inline site/docs/user_documentation/getting_started.md:78]:
```suggestion
> **Note**: The `SINGULARITY_*` variables used above can also be set user wide in your `.bashrc`. 
```

**Review by lelemmen — APPROVED** [review]
Great!

**Review by xdvriend — APPROVED** [review]
Everything looks very clear!

**Review by dariatols — APPROVED** [review]
Awesome!

**Pull request merged**